### ISSUE TITLE
Project on life support

### ISSUE THREAD

**Issue by lasseoe** [romkatv/powerlevel10k#2690]
I'm very sad to hear this :-(

What's the reason behind this decision?
Anything the community can do to help?

Thanks for a great theme/tool, it's changed how I work with the shell.

**Comment by romkatv** [comment]
Thanks for the kind words.

I want to reduce the amount of time I spend on support. The vast majority of issues that people file are essentially requests for personalized assistance. They are costly to me and do not result in any meaningful benefits to anyone other than the requestor. A small minority of issues are also mentally draining. Despite their small number, the effects are non-trivial.

The community can help by filing fewer issues: apply more effort trying to solve a problem before asking for help; do not suggest that the defaults must be changed to match one's needs exactly.

Note that powerlevel10k does not need ongoing active maintenance just to keep working. If it works for you now, it'll keep on working.

**Comment by johnalanwoods** [comment]
Thanks for all the work @romkatv!

**Comment by xeor** [comment]
Would it make sense to move the project to it's own organization and let the community take care of it? Make it p11k?
There arent a ton of people that have done recent changes, but based on the network graph, at least @jamebus and @lecodeski are ofte mergin in changes to their own fork.

**Comment by romkatv** [comment]
I won't be granting commit rights to the powerlevel10k repo to anyone else. If someone wishes to continue development, they can fork the project and proceed from there. This ensures that existing powerlevel10k users can trust my commitment to maintaining the stability and security of their prompts.

**Comment by akashdeepnandi** [comment]
thanks @romkatv, fan of your work.
I don't see it as any warning to not use the project anymore but as a guideline for most open source projects. After a point users should understand that a project sets sane defaults for 99% of the users. If you are an outlier, means you have the skillset or determination to go ahead, for the project and experiment as you wish.

**Comment by huyz** [comment]
> The vast majority of issues that people file are essentially requests for personalized assistance

I know not everyone can (we're all different and many of us are perfectionists/completionists), but I wish you could just do what many project owners do: ignore basic support requests or move them to the Discussions tab.
That way that might free up time for the critical things that few other people can do: write code for the project

**Comment by Syphdias** [comment]
I have resisted reacting to this issue for a while now. Because @romkatv said it all.

I would encourage everyone who loves this project to watch for „All activity“ in this project. Do this for two reasons:
1. See that you might not be fast enough to respond because „some guy“ (@romkatv) already postet the link to the relevant docs.
1. help people who have requests regarding config (could be an issue or a discussion topic). Chances are, you had similar issues at one point and figured it out or it is an interesting problem to solve.

For custom segments, or more complicated prompt behavior there are tools and documentation, if you want something custom.
If the documentation is not there yet, PR maybe?

TL;DR: this is a non-issue, do your part in supporting users and open source maintainers

**Comment by FranklinYu** [comment]
Rather than claiming p10k “in danger” I would rather call it “finished”: https://cassidoo.co/post/finished-apps/
